documents/MIGRATION_GUIDE.md

```diff
@@ -1,70 +1,66 @@
-    * [Client for Jobs Service](#client-for-jobs-service)
-    * [Client for Fleet Provisioning Service](#client-for-fleet-provisioning-service)
+    * [Client for AWS IoT Device Shadow](#client-for-device-shadow-service)
+    * [Client for AWS IoT Jobs](#client-for-jobs-service)
+    * [Client for AWS IoT fleet provisioning](#client-for-fleet-provisioning-service)
     * [Example](#example)
-* [How to Get Help](#how-to-get-help)
+* [How to get help](#how-to-get-help)
 * [Appendix](#appendix)
-    * [MQTT5 Features](#mqtt5-features)
-
-The V2 AWS IoT SDK for Java is a major rewrite of V1 SDK code base built on top of Java 8+. It includes many updates,
-such as improved consistency, ease of use, more detailed information about client status, an offline operation queue
-control, etc. This guide describes the major features that are new in V2 SDK, and provides guidance on how to migrate
-your code to V2 SDK from V1 SDK.
+    * [MQTT5 features](#mqtt5-features)
 
+## What's new in AWS IoT Device SDK for Java v2
 
-## What’s new in V2 SDK
-
-* V2 SDK client is truly async. Operations return `CompletableFuture` objects. Blocking calls can be emulated by waiting
+* The v2 SDK client is truly async. Operations return `CompletableFuture` objects. Blocking calls can be emulated by waiting
 for the returned `CompletableFuture` object to be resolved.
-* V2 SDK provides implementation for MQTT5 protocol, the next step in evolution of the MQTT protocol.
-* Public API terminology has changed. You `start()` or `stop()` the MQTT5 client rather than `Connect` or `Disconnect`
-like in V1. This removes the semantic confusion with the connect/disconnect as the client-level controls vs. internal
-recurrent networking events.
-* Support Jobs and Fleet Provisiong AWS IoT Core services.
-
-Public API for almost all actions and operations has changed significantly. For more details about the new features and
-to see specific code examples, refer to the other sections of this guide.
+* The v2 SDK provides implementation for MQTT 5 protocol, the next step in evolution of the MQTT protocol.
+* Public API terminology has changed. You `start()` or `stop()` the MQTT 5 client rather than `Connect` or `Disconnect`
+as in the v1 SDK. This removes the semantic confusion between client-level controls and internal recurrent networking events related to connection and disconnection.
+* The v2 SDK supports AWS IoT services such as Jobs and fleet provisioning.
 
+Public API for almost all actions and operations has changed significantly. For more information about the new features and specific code examples, refer to the [How to get started with v2 SDK](#how-to-get-started-with-v2-sdk) section of this guide.
 
-## How to Get Started with V2 SDK
+<!--Should this be public APIs? If it's plural, update the sentence like the following: Public APIs for almost all actions and operations have changed significantly. -->
```

Comment:
You're probably right, there are multiple APIs. I'll change this part after merging your PR.